lib/koala/api/batch_operation.rb

```diff
@@ -28,9 +28,15 @@ def initialize(options = {})
           raise AuthenticationError.new(nil, nil, "Batch operations require an access token, none provided.") unless @access_token
         end
 
-        def to_batch_params(main_access_token)
+        def to_batch_params(main_access_token, main_app_secret)
```

Comment:
We could probably call this just `app_secret` -- `main_access_token` is called that to distinguish it from the per-request access token. Since there isn't a per-request app secret,that isn't necesssary. (That said, I'm not sure if that'd be a less confusing or more confusing naming -- what do you think?)